### ISSUE TITLE
Unable to build locally 

### ISSUE THREAD

**Issue by mniewrzal** [mickaelistria/eclipse-bluesky#14]
I'm trying to build current master locally but I have error at the beginning of build:
```
[INFO] Resolving dependencies of MavenProject: org.eclipse.bluesky:repository:0.1.0-SNAPSHOT @ /home/wywrzal/git/eclipse-bluesky/repository/pom.xml
[INFO] {osgi.os=linux, osgi.ws=gtk, org.eclipse.update.install.features=true, osgi.arch=x86_64}
[ERROR] Cannot resolve project dependencies:
[ERROR]   Software being installed: repository raw:0.1.0.'SNAPSHOT'/format(n[.n=0;[.n=0;[-S]]]):0.1.0-SNAPSHOT
[ERROR]   Missing requirement: repository raw:0.1.0.'SNAPSHOT'/format(n[.n=0;[.n=0;[-S]]]):0.1.0-SNAPSHOT requires 'org.eclipse.mylyn.commons.core [3.22.0.v20161024-1634]' but it could not be found
[ERROR] 
[ERROR] See http://wiki.eclipse.org/Tycho/Dependency_Resolution_Troubleshooting for help.
[ERROR] Cannot resolve dependencies of MavenProject: org.eclipse.bluesky:repository:0.1.0-SNAPSHOT @ /home/wywrzal/git/eclipse-bluesky/repository/pom.xml: See log for details -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
```

**Comment by mickaelistria** [comment]
Mylyn bundles have changed symbolic-name. The category.xml file has to be updated accordingly.

**Comment by mickaelistria** [comment]
Should be fixed by bd65e077516495aa12f820d6dcbd2dcab856e6c5

**Comment by mniewrzal** [comment]
Works ok now, thanks.
